demo/site/src/blocks/TableBlock.tsx

```diff
@@ -0,0 +1,32 @@
+import { PropsWithData } from "@comet/cms-site";
+import { TableBlockData } from "@src/blocks.generated";
+
+// TODO: Used for debugging - either remove this or implement properly before merging feature branch
```

Comment:
Yes please

Reply:
Will be done in a follow-up PR, I'll keep this for now, as it's helpful during development.